playbooks/02 - Use Case - Phishing Email Triage Based On ML Classifier/Generate Phishing Email Alert.json

```diff
@@ -92,7 +95,7 @@
                 },
                 "resource": {
                     "name": "{% if vars.item.headers.subject %}Phishing Email: {{vars.item.headers.subject}} {% else %} None {% endif %}",
-                    "type": "\/api\/3\/picklists\/0b3ef6f9-eb29-4ab9-ac98-98364bd1a3aa",
+                    "type": "\/api\/3\/picklists\/c145394b-069a-449b-a05e-67980d0f4b82",
```

Comment:
what is this change all about?

Reply:
Here, this is related to Mantis #0857499 where a playbook is modified to create a demo alert with the alert type 'Suspicious Mail'. Previous it was created as a 'Phishing' alert type.